api/app/Http/Controllers/Api/AuthController.php

```diff
@@ -77,4 +81,75 @@ public function register(UserRegistrationModel $request)
 
         User::create($account);
     }
+
+    /**
+     * Send a mail for resetting the password.
+     *
+     * @param \App\Http\Requests\Api\PasswordResetModel $request
+     *
+     * @return mixed
+     */
+    public function sendResetMail(PasswordResetModel $request)
+    {
+        $user_email = $request->email;
+        $user = User::where('email', $user_email)->first();
+        if ($user && $this->notSpamming($user, env('PASSWORD_RESET_MINUTES', 30))) {
```

Comment:
Can you use isSpamming instead of notSpamming, it's a convention

Don't use env but use instead config. See why: 
https://laravel.com/docs/5.4/configuration#configuration-caching